package.json

```diff
@@ -53,7 +52,6 @@
     "ethers": "^5.6.1",
```

Comment:
Is bunyan-debug-stream development-only?

Reply:
Yeah it's just. for logging